modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubTextLayout.java

```diff
@@ -92,7 +92,17 @@ public BaseBounds getBounds() {
 
     @Override
     public BaseBounds getBounds(TextSpan filter, BaseBounds bounds) {
-        final double fontSize = (font == null ? nullFontSize : font.getSize());
+        double fontSize = nullFontSize;
+        if (font != null) {
+            fontSize = font.getSize();
+
+            // For better testing, we make bold text a little bit bigger.
+            boolean bold = font.getStyle().toLowerCase().contains("bold");
+            if (bold) {
+                fontSize++;
```

Comment:
should this only apply to width computation (line 128)?

in other words, the height of for bold text of size 12 should have the same height as plain text size 12, shouldn't it?

Reply:
good question! I checked some fonts and they do look to me that they have the same height regular as well as bold.
So it's probably better to use the normal size for the height here?